canonical-web-and-design/microk8s.io
# Update homepage to match latest copy

## Done

Update homepage to match latest copy

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8027/
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Compare to [copy](https://docs.google.com/document/d/1ObWmhRc4GCIhrBi8gT7yIIs0J_dn_8ravfLtcwhr3uQ/edit#)


## Issue / Card

Fixes https://github.com/canonical-web-and-design/web-squad/issues/2674

## Screenshots

![image](https://user-images.githubusercontent.com/40214246/81057203-e2058180-8ec3-11ea-8a2e-ccb250383a7d.png)]

![image](https://user-images.githubusercontent.com/40214246/81057249-f8abd880-8ec3-11ea-9ffd-f5fbd8b52c86.png)
